Vagrantfile

```diff
@@ -12,6 +12,69 @@ Vagrant.configure("2") do |config|
   # using a specific IP.
   config.vm.network "private_network", ip: "192.168.127.127"
 
+
+  # vb.customize does not returns output from the command
+  id_file = ".vagrant/machines/default/virtualbox/id"
+  # don't know the id, can't search for the controller name
+  # using the default on the config.vm.box and hoping for the best
+  floppy_controller = "Floppy Controller Controller"
+  has_cdrom = false
+
+  # doesn't exist until VM is provisioned the first time
+  if FileTest.exist?(id_file) and FileTest.readable?(id_file)
+    # unsetting, if nothing is found no need to remove again
+    floppy_controller = ""
+    id = File.read(id_file)
+    output = `VBoxManage showvminfo #{id}`
+    previous_line = ''
+
+    for line in output.each_line
+      if line =~ /I82078/
+        floppy_controller = (previous_line.gsub(/\s/, '').split(':'))[1]
+      elsif line =~ /^SATA\sController\s\(1,\s0\)\:\sEmpty/
+          has_cdrom = true
+      else
+        previous_line = line
+      end
+    end
+  end
+
+  config.vm.provider "virtualbox" do |vb|
+     # Display the VirtualBox GUI when booting the machine
+     # Might be useful, but not required, so leaving commented
+     # vb.gui = true
+     # Customize the amount of memory on the VM:
+     vb.memory = "1024"
+     # Giving a nice name to it
+     vb.name = "CPAN Testers - Development"
+     # see https://www.virtualbox.org/manual/ch08.html
+     # configure the minimum memory for video (set to 8MB originally)
+     vb.customize ["modifyvm", :id, "--vram", "9"]
+     # This needs to be set as idempotent change
+     # remove the floppy
+     unless floppy_controller.empty?
+       vb.customize ["storagectl", :id, "--name", floppy_controller, "--remove"]
+     end
+     # adds a CDROM
+     # also assumes config.vm.box has a single SATA HD in port 0
+     unless has_cdrom
+       vb.customize ["storageattach", :id, "--storagectl", "SATA Controller", "--port", "1", "--medium", "emptydrive"]
+     end
+  end
+
+  # Guest Additions handling
+  config.vm.provision "shell", inline: "apt-get update;apt-get -y upgrade;apt-get -y install module-assistant;m-a prepare --text-mode --non-inter"
```

Comment:
I did a quick comparison (using `dpkg-query`):

```
$ diff -y all_prod.txt shared/all_vm.txt | head -30
3ware-tools 10.2-2					      <
ack-grep 2.14-4							ack-grep 2.14-4
acl 2.2.52-2							acl 2.2.52-2
acpi 1.7-1							acpi 1.7-1
acpid 1:2.0.23-2						acpid 1:2.0.23-2
acpi-support-base 0.142-6					acpi-support-base 0.142-6
adduser 3.113+nmu3						adduser 3.113+nmu3
amd64-microcode 2.20141028.1				      |	apache2 2.4.10-10+deb8u8
apache2 2.4.10-10+deb8u1				      |	apache2-bin 2.4.10-10+deb8u8
apache2-bin 2.4.10-10+deb8u1				      |	apache2-data 2.4.10-10+deb8u8
apache2-data 2.4.10-10+deb8u1				      |	apache2-utils 2.4.10-10+deb8u8
apache2-utils 2.4.10-10+deb8u1				      |	apt 1.0.9.8.4
apt 1.0.9.8						      |	aptitude 0.6.11-1+b1
apt-file 2.5.4						      |	aptitude-common 0.6.11-1
apt-utils 1.0.9.8					      |	aptitude-doc-en 0.6.11-1
atop 1.26-2						      |	apt-listchanges 2.85.13+nmu1
base-files 8+deb8u1					      |	apt-utils 1.0.9.8.4
							      >	at 3.1.16-1
							      >	base-files 8+deb8u7
base-passwd 3.5.37						base-passwd 3.5.37
bash 4.3-11+b1						      |	bash 4.3-11+deb8u1
bind9-host 1:9.9.5.dfsg-9+deb8u2			      |	bash-completion 1:2.1-4
							      >	bc 1.06.95-9
							      >	bind9-host 1:9.9.5.dfsg-9+deb8u10
binutils 2.25-5							binutils 2.25-5
							      >	bsd-mailx 8.1.2-0.20141216cvs-2
bsdmainutils 9.0.6						bsdmainutils 9.0.6
bsdutils 1:2.25.2-6						bsdutils 1:2.25.2-6
build-essential 11.7						build-essential 11.7
busybox 1:1.22.0-9+deb8u1					busybox 1:1.22.0-9+deb8u1
```

Not very easy to visualize here, I just attached the whole file.
[output.txt](https://github.com/cpan-testers/cpantesters-deploy/files/959039/output.txt)

I guess we should generate the list of packages from the production server and reproduce it on the VM, even though versions wouldn't match. After some iterations of testing in the VM, we could work on some scheduled updates on production as well.

Let me know if you want to pursue this, I can work through to automate this step on the VM.